lib/ansible/modules/database/misc/elasticsearch_plugin.py

```diff
@@ -41,12 +41,12 @@
         description:
             - "Timeout setting: 30s, 1m, 1h..."
         required: False
-        default: 1m
+        default: None
     plugin_bin:
         description:
             - Location of the plugin binary
         required: False
-        default: /usr/share/elasticsearch/bin/plugin
+        default: /usr/share/elasticsearch/bin/elasticsearch-plugin
```

Comment:
you should note that the default changes in 2.4 and why

Reply:
Rather than change the default, the module needs to account for the different binary name of different Elasticsearch versions. I have a [version of this module that does that](https://github.com/samdoran/ansible-role-elasticsearch/blob/master/library/elasticsearch_plugin.py). I'll clean it up and submit a PR.